tests/end_to_end/helpers/assertions.py

```diff
@@ -172,13 +172,14 @@ def assert_row_counts_equal(tap_query_runner_fn: callable, target_query_runner_f
 # pylint: disable=too-many-locals
 def assert_all_columns_exist(tap_query_runner_fn: callable,
                              target_query_runner_fn: callable,
-                             colum_type_mapper_fn: callable = None) -> None:
+                             column_type_mapper_fn: callable = None,
+                             ignore_cols: Union[Set, List] = None) -> None:
     """Takes two query runner methods, gets the columns list for every table in both the
     source and target database and tests if every column in source exists in the target database.
 
     :param tap_query_runner_fn: method to run queries in the first connection
     :param target_query_runner_fn: method to run queries in the second connection
-    :param colum_type_mapper_fn: method to convert source to target column types"""
+    :param column_type_mapper_fn: method to convert source to target column types"""
```

Comment:
can we document `ignore_cols` as well?

Reply:
Documented in https://github.com/transferwise/pipelinewise/pull/495/commits/f039985bb3223703604e406f78f5c9cf39ab696b.
Let me if it needs more clarification.